openml/runs/functions.py

```diff
@@ -296,7 +294,7 @@ def _run_task_get_arffcontent(model, task, class_labels):
     user_defined_measures = defaultdict(lambda: defaultdict(dict))
 
     rep_no = 0
-    can_measure_runtime = version_complies(3, 3) and _check_n_jobs(model)
+    can_measure_runtime = sys.version_info[:2] >= (3, 3) and _check_n_jobs(model)
```

Comment:
can you maybe add a short comment on the sys.version_info? If I were to look at the code, this would seem magic to me. Thanks!